doc-content/enterprise-only/integration/sso-remote-services-securing-proc.adoc

```diff
@@ -1,7 +1,7 @@
 [id='sso-remote-services-securing-proc']
-= Securing {CENTRAL} file system services using RH-SSO
+= Enabling access to external file system and Git repository services for {CENTRAL} using RH-SSO
```

Comment:
[Style] Depending on the relation between these, suggest that "system" is pluralized or the title rephrased if external services applies to file system and git repositories. Examples:

Enabling access to external file systems and Git repository services with {CENTRAL} by using RH-SSO
Enabling access to external services such as file systems and Git repositories by using RH-SSO
Accessing external file systems and Git repository services with {CENTRAL} by using RH-SSO